tests/cephadm/test_client.py

```diff
@@ -124,6 +124,12 @@ def setup(host):
                 for repos in cdn_ceph_repo[rhel_version][rhcs_version]:
                     _node.exec_command(sudo=True, cmd=f"{enable_cmd}{repos}")
 
+                # Clearing the release preference set and cleaning all yum repos
+                # Observing selinux package dependency issues for ceph-base
+                wa_cmds = ["subscription-manager release --unset", "yum clean all"]
+                for wa_cmd in wa_cmds:
+                    _node.exec_command(sudo=True, cmd=wa_cmd)
```

Comment:
Is it possible to put this in a conditional block for only `ceph-base` installation, this could reduce ripple effect, if any.

Reply:
I do not see any issues here as this is run only when we have Custom CDN repos added.
Without yum clean, older version that was already present could affect the version of the packages.